docs/en/guides/sahi-tiled-inference.md

```diff
@@ -99,8 +99,8 @@ You can instantiate a YOLO11 model for object detection like this:
 from sahi import AutoDetectionModel
 
 detection_model = AutoDetectionModel.from_pretrained(
-    model_type="yolov8",
-    model_path=yolov8_model_path,
+    model_type="yolo11",
```

Comment:
@RizwanMunawar passing this var as `ultralytics` also works, this way its possible to put any ultralytics model into `model_path` 👍🏻 

```suggestion
    model_type="ultralytics",
```

Reply:
@fcakyon Wow, that's nice! It will work for YOLOv8 too! Than?

Reply:
yes @RizwanMunawar 🔥